src/test/java/com/exasol/edmlgenerator/parquet/ParquetColumnToMappingDefinitionConverterTest.java

```diff
@@ -47,4 +110,96 @@ private void assertConvertsToToDecimalMapping(final Type type, final int scale,
+                .convert(logicalTimestampType);
+        assertAll(//
+                () -> assertThat(toTimestampMapping.getDestinationName(), equalTo("MY_TIMESTAMP")), //
+                () -> assertThat(toTimestampMapping.isUseTimestampWithLocalTimezoneType(), equalTo(true))//
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("testConvertToStringCases")
+    void testConvertToString(final LogicalTypeAnnotation annotation) {
+        final Type logicalStringType = Types.primitive(BINARY, REQUIRED).as(annotation).named("value");
+        assertConvertsToToVarcharMapping(logicalStringType, "VALUE", 2_000_000);
+    }
+
+    private void assertConvertsToToVarcharMapping(final Type type, final String expectedName, final int expectedSize) {
+        final ToVarcharMapping toVarcharMapping = (ToVarcharMapping) new ParquetColumnToMappingDefinitionConverter()
+                .convert(type);
+        assertAll(//
+                () -> assertThat(toVarcharMapping.getVarcharColumnSize(), equalTo(expectedSize)),
+                () -> assertThat(toVarcharMapping.getDestinationName(), equalTo(expectedName))//
+        );
+    }
+
+    @Test
+    void testConvertList() {
+        final Type itemType = Types.primitive(INT32, REQUIRED).named("element");
+        final GroupType listType = Types.list(REQUIRED).element(itemType).named("ids");
+        System.out.println(listType.toString());
+        assertConvertsToJsonMapping(listType, "IDS");
+    }
+
+    @Test
+    void testConvertMap() {
+        final Type keyType = Types.primitive(BINARY, REQUIRED).as(LogicalTypeAnnotation.stringType()).named("key");
+        final Type valueType = Types.primitive(INT32, REQUIRED).named("value");
+        final GroupType listType = Types.map(REQUIRED).key(keyType).value(valueType).named("scores");
+        System.out.println(listType.toString());
+        assertConvertsToJsonMapping(listType, "SCORES");
+    }
+
+    @Test
+    void testConvertEnum() {
+        final Type keyType = Types.primitive(BINARY, REQUIRED).as(LogicalTypeAnnotation.stringType()).named("key");
+        final Type valueType = Types.primitive(INT32, REQUIRED).named("value");
+        final GroupType listType = Types.map(REQUIRED).key(keyType).value(valueType).named("scores");
+        System.out.println(listType.toString());
+        assertConvertsToJsonMapping(listType, "SCORES");
+    }
+
+    private void assertConvertsToJsonMapping(final GroupType listType, final String expectedName) {
+        final ToJsonMapping toJsonMapping = (ToJsonMapping) new ParquetColumnToMappingDefinitionConverter()
+                .convert(listType);
+        assertAll(//
+                () -> assertThat(toJsonMapping.getVarcharColumnSize(), equalTo(2_000_000)),
+                () -> assertThat(toJsonMapping.getDestinationName(), equalTo(expectedName))//
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("getUnsupportedTypes")
+    void testUnsupported(final Type unsupportedType) {
```

Comment:
Put `getUnsupportedTypes` next to this test

Reply:
same